recipes/pyLBL/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set name = "pyLBL" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/GRIPS-code/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 010e8b395e8ff6f05c551727e579a7852a7e3808a1c2be47797fa3c2335efbe2
+
+build:
+  skip: true  # [py<36]
+  skip: true  # [win]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - python
+    - sqlite
+  host:
+    - python
+    - pip
+    - setuptools
+    - sqlite
+  run:
+    - python
+    - netCDF4
+    - numpy
+    - scipy
+    - sqlalchemy
+    - xarray
+    - {{ pin_compatible("sqlite") }}
+
+test:
+  imports:
+    - pyLBL
+  requires:
+    - pytest
+  source_files:
+    - tests/conftest.py
+    - tests/test_artscrossfit.py
+    - tests/test_atmosphere.py
+    - tests/test_mt_ckd.py
+    - tests/test_plugins.py
+    - tests/test_tips.py
+  commands:
+    - pytest tests --ignore=tests/test_database.py --ignore=tests/test_gas_optics.py --ignore=tests/test_spectroscopy.py --ignore=tests/test_webapi.py
+
+about:
+  home: https://github.com/GRIPS-code/{{ name }}
+  summary: 'A python package for calculating line-by-line molecular spectra'
+  description: |
+    pyLBL provides a python interface to line-by-line calculations of absorption
+    coefficients that is traceable to benchmarks. The overall goal is to treat lines,
+    cross-sections, continua, collision-induced absorption, and line mixing. It is
+    written in Python 3.5+ and includes a C extenstion module by default.
+  license: LGPL-2.1-only
+  license_family: LGPL
+  license_file: LICENSE.md
+  doc_url: https://pylbl-1.readthedocs.io/en/first-release
+  dev_url: https://github.com/GRIPS-code/{{ name }}
```

Better not to template the name here so that the URLs can more easily be copy/pasted from the recipe:
```suggestion
  home: https://github.com/GRIPS-code/pyLBL
  summary: 'A python package for calculating line-by-line molecular spectra'
  description: |
    pyLBL provides a python interface to line-by-line calculations of absorption
    coefficients that is traceable to benchmarks. The overall goal is to treat lines,
    cross-sections, continua, collision-induced absorption, and line mixing. It is
    written in Python 3.5+ and includes a C extenstion module by default.
  license: LGPL-2.1-only
  license_family: LGPL
  license_file: LICENSE.md
  doc_url: https://pylbl-1.readthedocs.io/en/first-release
  dev_url: https://github.com/GRIPS-code/pyLBL
```

---

Fixed, thank you.